front/app/components/FilterBoxes/StatusFilter.tsx

```diff
@@ -6,7 +6,9 @@ import {
   defaultCardStyle,
   isRtl,
```

Comment:
Just used the CollapsibleContainer in this component as well, so when we release the Sorting Box the UI will match the existing filters boxes. 

We can fix the styling/clean up code in this file more when we work on the Status Filter ticket :)